src/lib/zork/index.ts

```diff
@@ -287,7 +349,7 @@ const onSocksConnection = (ctx: Context, sessionId: any) => {
       return this;
     },
     handleDisconnect: () => {
-      channel.close();
+      //channel.close();  // TODO need this?
```

Comment:
I'm not sure what's causing the segfaults. Improved the comment though in 66a1b24. Still haven't seen any segfaults now that we're not closing datachannels from the give side. If I see them again though I'll look into what's causing them.